### PR TITLE
Feature/nex 150 update code from old repo

### PR DESCRIPTION
**Pull request by btamas**
Jira: https://oat-sa.atlassian.net/browse/NEX-150

Update code from extension-tao-testqti

Test: npm run test

Requirement:
- [x] https://github.com/oat-sa/tao-test-runner-fe/pull/3
- [x] Add the version of `@oat-sa/tao-test-runner` to `package.json`

Related PR:
- [ ] https://github.com/oat-sa/extension-tao-testqti/pull/1542

Task after merge:
- [ ] Release it to npm

### CODE REVIEW

File: src/provider/qti.js [hunk 1]
```diff
@@ -69,7 +69,7 @@ var qtiProvider = {
     loadProxy: function loadProxy() {
```
Comment (ferenckiss89) [inline src/provider/qti.js]: update the date in the header please

File: src/proxy/cache/proxy.js [hunk 2]
```diff
@@ -51,8 +51,11 @@ var loadNextDelay = 450;
  */
```
Comment (ferenckiss89) [inline src/proxy/cache/proxy.js]: update the date in header please

File: src/proxy/loader.js [hunk 3]
```diff
@@ -17,6 +17,9 @@
  */
```
Comment (ferenckiss89) [inline src/proxy/loader.js]: update the date in header please

File: test/proxy/offline/proxy/test.js [hunk 4]
```diff
@@ -177,7 +177,7 @@ define([
                             }
```
Comment (ferenckiss89) [inline test/proxy/offline/proxy/test.js]: update the date in header please

File: package.json [hunk 5]
```diff
@@ -40,23 +40,23 @@
...
+        "@oat-sa/tao-core-libs": "^0.1.1",
+        "@oat-sa/tao-core-sdk": "^0.4.3",
+        "@oat-sa/tao-core-ui": "^0.16.4",
         "@oat-sa/tao-item-runner": "^0.3.0",
-        "@oat-sa/tao-item-runner-qti": "^0.3.0",
-        "@oat-sa/tao-qunit-testrunner": "^0.1.2",
-        "@oat-sa/tao-test-runner": "^0.1.0",
+        "@oat-sa/tao-item-runner-qti": "^0.3.1",
+        "@oat-sa/tao-qunit-testrunner": "^0.1.3",
+        "@oat-sa/tao-test-runner": "oat-sa/tao-test-runner-fe#feature/NEX-150_update-code-from-old-repo",
```
Comment (jsconan) [inline package.json]: Note for myself: replace that upon merge.

**Pull request merged**